api/src/main/java/org/pmiops/workbench/api/InstitutionController.java

```diff
@@ -103,7 +114,9 @@ private Institution getInstitutionImpl(final String shortName) {
                 () ->
                     new NotFoundException(
                         String.format("Could not update Institution '%s'", shortName)));
-
+    institutionService
+        .getAffiliatedUsers(shortName)
+        .forEach(u -> userService.updateUserAccessTiers(u, Agent.asAdmin(userProvider.get())));
```

Comment:
It might be more natural for `InstitutionService` method to call this method instead of `InstitutionController`, but it is often difficult for another service to include UserService.  Here's it's pretty easy.

Reply:
I made an attempt but I abandoned it because I couldn't make the Spring dependencies work for LoadInstitutions.

Reply:
can institution short name  be updated by user?

Reply:
unlikely. b/c it's `@AuthorityRequired({Authority.INSTITUTION_ADMIN})`

Reply:
It can be updated by the admin, if that's what you're asking.

We don't expose this in the UI, however.

Reply:
Per Brian's suggestion, I'm removing this ability: #6052